docs/api-reference/indexing/create_index.mdx

```diff
@@ -0,0 +1,316 @@
+  record types.
+</ParamField>
+<ParamField body="normalizer">
+  The name of the tokenizer used for fast fields. This field is ignored unless
+  `fast=true`. See [normalizers](/api-reference/indexing/fast_fields#normalizers) for a list of available
+  normalizers.
+</ParamField>
+
+
+### Datetime Fields
+
+Columns of type `DATE`, `TIMESTAMP`, `TIMESTAMPTZ`, `TIME`, `TIMETZ`, and their corresponding array types can be indexed as `datetime_fields`.
+Indexing a datetime field is useful if it is used for filtering as part of the search query.
+
+```sql
+CALL paradedb.create_bm25(
+  index_name => 'search_idx',
+  table_name => 'mock_items',
+  key_field => 'id',
+  datetime_fields => paradedb.field('created_at')
+);
+```
+
+`paradedb.field` accepts several configuration options for boolean fields:
+
+<ParamField body="indexed" default={true}>
+  Whether the field is indexed. Must be `true` in order for the field to be
+  tokenized and searchable.
+</ParamField>
+<ParamField body="stored" default={true}>
+  Whether the original value of the field is stored.
+</ParamField>
+<ParamField body="fast" default={true}>
+  Fast fields can be random-accessed rapidly. Fields used for aggregation must
+  have `fast` set to `true`. Fast fields are also useful for accelerated scoring
+  and filtering.
+</ParamField>
+
+### Array Fields
+
+`text_fields`, `numeric_fields`, `boolean_fields`, and `datetime_fields` support array types. For instance, columns of
+type `TEXT[]` can be passed into `text_fields`, and columns of type `INT[]` can be passed into `numeric_fields`.
+
+<Note>
+The only exception are `JSON[]` and `JSONB[]` types, which are not yet supported.
+</Note>
+
+### Multiple Fields
+
+The `||` operator can be used to index multiple fields.
+
+```sql
+CALL paradedb.create_bm25(
+  index_name => 'search_idx',
+  table_name => 'mock_items',
+  key_field => 'id',
+  text_fields => paradedb.field('description') || paradedb.field('category')
+);
+```
+
+## Choosing a Key Field
+
+The `key_field` option is used to uniquely identify documents within an index and cannot be tokenized. For instance,
+the following configuration is not allowed:
+
+```sql
+-- This will throw an error
+CALL paradedb.create_bm25(
+  index_name => 'search_idx',
+  table_name => 'mock_items',
+  key_field => 'description',
+  text_fields => paradedb.field('description')
+);
+```
+
+## Partial Index
+
+The following code block demonstrates how to pass predicates to `create_bm25`
+to construct a [partial index](https://www.postgresql.org/docs/current/indexes-partial.html). Partial
+indexes are useful for reducing index size on disk and improving update speeds over non-indexed rows.
+
+```sql
+CALL paradedb.create_bm25(
+  index_name => 'search_idx',
+  table_name => 'mock_items',
+  key_field => 'id',
+  text_fields => paradedb.field('description'),
+  predicates => 'category = ''Electronics'' AND rating > 2'
+);
+```
+
+## Partitioned Index
```

Comment:
Thoughts on making "Create Index" a top-level category and make "Partial, Partitioned, etc." be top-level pages within that?

Reply:
I like the existing structure better.